go/tools/builders/stdliblist_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+)
+
+func Test_stdliblist(t *testing.T) {
+	testDir := t.TempDir()
+	f, _ := ioutil.TempFile(testDir, "*")
+
+	// test files are at run file directory, but this test is run at
+	// {runfile directory}/bazel.TestWorkspace()
+	// since -sdk is assumed to be a relative path to execRoot
+	// (go.sdk.root_file.dirname), thus setting wd to
+	// {runfile directory} so that go_sdk is discoverable
+	// {runfile directory} is the parent directory of bazel.RunfilesPath()
+	runFilesPath, err := bazel.RunfilesPath()
+	if err != nil {
+		t.Error("failed to find runfiles path")
+	}
+	test_args := []string{
+		fmt.Sprintf("-out=%s", f.Name()),
+		fmt.Sprintf("-sdk=%s", "go_sdk"),
```

Comment:
Can you pass the absolute path to go_sdk here? It should be available in `$TEST_SRCDIR/external/go_sdk`. With the absolute path, you don't need to pass `-wd`, right?

Reply:
`-sdk` passed by `go.sdk.root_file.dirname` in stdlib.bzl, which is a relative path. See below comment